app/sagas/init.js

```diff
@@ -14,9 +13,13 @@ import database from '../lib/realm';
 
 const restore = function* restore() {
 	try {
+		const { serversDB } = database.databases;
+
+		const currentServer = serversDB.objects('servers').filtered('currentServer = true');
```

Comment:
do `const { userToken: token, id: server } = yield serversDB.objects('servers').filtered('currentServer = true')[0];` here and remove next lines where you are defining `{token, server}`